src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -142,18 +152,18 @@ public void onUpdate(@NotNull TabPlayer tabPlayer) {
 
     @NotNull
     public Component getHeader(@NotNull TabPlayer player) {
-        return new MineDown(Placeholder.format(plugin.getSettings().getHeader(), plugin, player)).toComponent();
+        return new MineDown(Placeholder.format(plugin.getSettings().getHeader(plugin.getSettings().getServerGroup(player.getServerName())), plugin, player)).toComponent();
```

Comment:
this is quite a long single-line, maybe break after the second `plugin.getSettings()` call.